src/jit/morph.cpp

```diff
@@ -12270,7 +12270,7 @@ GenTreePtr Compiler::fgMorphSmpOp(GenTreePtr tree, MorphAddrContext* mac)
                     fgMorphTreeDone(op2);
 
                     oper = GT_ADD;
-                    tree->ChangeOper(oper);
+                    tree->ChangeOper(oper, GenTree::PRESERVE_VN);
```

Comment:
Not clear that preserving this VN is useful, as the child node is a new GT_NEG which will not have a VN
(It will default to NoVN)